websites/I/iQIYi/dist/metadata.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "SlowLife",
+      "id": "374905512661221377"
+    },
+    "url": "www.iq.com",
+    "description": {
+      "en": "iQIYI is the world's leading online movie and video streaming website, offering TV dramas, movies, variety shows, animation, and other quality content.",
+      "zh_TW": "愛奇藝-全球領先的線上影音視頻網站-正版高清影片影視免費線上看"
+    },
+    "service": "iQIYI",
```

Comment:
server and folder name need to be the exact same

Reply:
Done